### PR TITLE
[SPARK-30085][SQL][DOC] Standardize sql reference

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Standardize sql reference

### Why are the changes needed?
To have consistent docs 

### Does this PR introduce any user-facing change?
Yes

### How was this patch tested?
Tested using jykyll build --serve

### CODE REVIEW

**Comment by huaxingao** [comment]
I know there are other inconsistencies. I will fix one by one.

**Review by srowen — COMMENTED** [review]
Looks directionally fine, ping when you're ready.

**Comment by huaxingao** [comment]
@srowen @dilipbiswal 
I have the following changes:
1. use the same ```table_identifier``` syntax for all the commands that use ```table_identifier```
2. use the same ```view_identifier``` syntax for all the commands that use ```view_identifier```
3. use the same ```partition_spec``` syntax for all the commands that use ```partition_spec```
4. some of the commands have a space in between of the braces and text. Some of them don't. I make it consistent to have a space
5. remove ; from the end of the sql syntax

There may be other inconsistencies. If I find more later, I will open a separate PR to fix the problems.

File: docs/sql-ref-syntax-aux-cache-cache-table.md [hunk 1]
```diff
@@ -35,8 +35,14 @@ CACHE [ LAZY ] TABLE table_name
 </dl>
 
 <dl>
-  <dt><code><em>table_name</em></code></dt>
-  <dd>The name of the table to be cached.</dd>
+  <dt><code><em>table_identifier</em></code></dt>
+  <dd>
+    Specifies a table name, which may be optionally qualified with a database name.<br><br>
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-cache-cache-table.md]: @huaxingao Here, can we try and retain the original description ? How about : 
```
Specifies the table name to be cached. A table name may be optionally qualified with a database name.
```

File: docs/sql-ref-syntax-aux-cache-uncache-table.md [hunk 2]
```diff
@@ -24,12 +24,18 @@ license: |
...
-UNCACHE TABLE [ IF EXISTS ] table_name
+UNCACHE TABLE [ IF EXISTS ] table_identifier
 {% endhighlight %}
 ### Parameters
 <dl>
- <dt><code><em>table_name</em></code></dt>
- <dd>The name of the table or view to be uncached.</dd>
+  <dt><code><em>table_identifier</em></code></dt>
+  <dd>
+    Specifies a table name, which may be optionally qualified with a database name.<br><br>
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-cache-uncache-table.md]: @huaxingao Same comment as above.

File: docs/sql-ref-syntax-aux-refresh-table.md [hunk 3]
```diff
@@ -26,17 +26,17 @@ lazy manner when the cached table or the query associated with it is executed ag
...
+REFRESH [TABLE] table_identifier
 {% endhighlight %}
 
 ### Parameters
 <dl>
-  <dt><code><em>tableIdentifier</em></code></dt>
+  <dt><code><em>table_identifier</em></code></dt>
   <dd>
-    Specifies a table name, which is either a qualified or unqualified name that designates a table/view. If no database identifier is provided, it refers to a temporary view or a table/view in the current database.<br><br>
+    Specifies a table name, which may be optionally qualified with a database name.<br><br>
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-refresh-table.md]: @huaxingao ditto as above..

File: docs/sql-ref-syntax-aux-show-partitions.md [hunk 4]
```diff
@@ -26,19 +26,30 @@ partition spec.
...
+      </code>
+  </dd>
 </dl>
 <dl>
-  <dt><code><em>PARTITION ( partition_col_name [ = partition_col_val ] [ , ... ] )</em></code></dt>
-  <dd>An optional parameter that specifies a comma separated list of key and value pairs for
-      partitions. When specified, the partitions that match the partition spec are returned.</dd>
+  <dt><code><em>partition_spec</em></code></dt>
+  <dd>
+    An optional parameter that specifies a comma separated list of key and value pairs
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-show-partitions.md]: @huaxingao ditto as above.

File: docs/sql-ref-syntax-aux-show-table.md [hunk 5]
```diff
@@ -50,10 +51,14 @@ SHOW TABLE EXTENDED [IN|FROM database_name] LIKE 'identifier_with_wildcards' [PA
        <li> The leading and trailing blanks are trimmed in the input pattern before processing.</li>
     </ul> 
   </dd>
-  <dt><code><em>PARTITION(partition_spec)</em></code></dt>
+  <dt><code><em>partition_spec</em></code></dt>
   <dd>
-    Specifies partition column and its value which is exists in the table. Note that a table regex 
-    cannot be used with a partition specification..
+    An optional parameter that specifies a comma separated list of key and value pairs
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-show-table.md]: @huaxingao here there is a mention about mutual exclusivity between the LIKE pattern and partition spec. Can we mention it here as well ?

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 6]
```diff
@@ -118,26 +127,39 @@ exsisting tables.
...
+      </code>
+  </dd>
 </dl>
 
 <dl>
-  <dt><code><em>PARTITION (part_spec)</em></code></dt>
-  <dd>Specifies the partition on which the property has to be set.</dd>
+  <dt><code><em>partition_spec</em></code></dt>
+  <dd>
+    An optional parameter that specifies a comma separated list of key and value pairs
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-alter-table.md]: @huaxingao same ..

**Comment by dilipbiswal** [comment]
@huaxingao Looks good to me. Please take a look at some minor comments.
Thanks a lot for doing it.

**Comment by dilipbiswal** [comment]
LGTM

**Comment by srowen** [comment]
Merged to master

**Comment by huaxingao** [comment]
Thanks! @srowen @dilipbiswal

**Pull request closed without merging**